packages/amplify_datastore/android/src/main/kotlin/com/amazonaws/amplify/amplify_datastore/types/FlutterDataStoreFailureMessage.kt

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amazonaws.amplify.amplify_datastore.types
+
+enum class FlutterDataStoreFailureMessage {
+    CASTING {
+        override fun toString(): String {
+            return "ERROR_CASTING_INPUT_IN_PLATFORM_CODE"
+        }
+    },
+    MALFORMED {
+        override fun toString(): String {
+            return "AMPLIFY_REQUEST_MALFORMED"
+        }
+    },
+    QUERY {
+        override fun toString(): String {
+            return "AMPLIFY_DATASTORE_QUERY_FAILED"
+        }
```

Comment:
Nit: Why not get rid of the override and toString() for each message and just do:
```
enum class FlutterDataStoreFailureMessage {
ERROR_CASTING_INPUT_IN_PLATFORM_CODE,
AMPLIFY_REQUEST_MALFORMED,
AMPLIFY_DATASTORE_QUERY_FAILED,
}
```

Reply:
Taken from how it's done in Auth plugin. @haverchuck any reason to not go with this approach?

Reply:
@Amplifiyer If this works, I say go for it.

Reply:
It should work, it's just that the enum constants will grow longer which I believe is fine.